pymodbus/datastore/remote.py

```diff
@@ -38,25 +39,37 @@ def validate(self, _fc_as_hex, _address, _count):
         """
         return True
 
-    def getValues(self, fc_as_hex, _address, _count=1):
+    async def async_getValues(self, fc_as_hex, address, count=1):
```

Comment:
I left these changes as separate commit, incase there is some desire to keep this still intact.

Reply:
it is the other way around, we do not want the cascade changes your original PR contained and still contains.

Having the async_getValues, should solve your problem, so please revert all the cascaded changes in the system (your PR still touches 31 files).